### ISSUE TITLE
The object.objectType property

### ISSUE THREAD

**Issue by smnmyr** [dret/exercise#4]
object.objectType could be used to differentiate between different representations of the same object (or even different objects) that all relate to the same activity, e.g. an image of a running track vs. the HRM data

**Comment by dret** [comment]
agreed. also, the question is whether it's good or bad that in the first attempt at #1, the object type and the verb are the same. maybe it would be better to have the verb as "exercise" and the object type as something like "exerciseRecord", because the object representing the actual activity is the activity itself?
